data/remnant/remnant logistics.txt

```diff
@@ -0,0 +1,295 @@
+	cargo  "remnant industrial" 30 300
+	job
+	repeat
+	to offer
+			not "Remnant: Ssil Vida Active"
+			not "flagship planet: Ssil Vida"
+			has "Remnant: Logistics 1: done"
+			has "Remnant: Logistics Civil Engineering: done" >= 9
+			random < 76
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A project technician inspects the <cargo> and approves your payment of <payment>.`
+		payment 55000 15
+
+trade	
+	commodity "remnant outfitter repair"
+		"Inhibitor Cannon"
+		"Thrasher Cannon"
+		"Thrasher Turret"
+		"Point Defense Turret"
+		"EMP Torpedo Storage Bay"
+		"EMP Torpedo Bay"
+		"Millennium Cell"
+		"Epoch Cell"
+		"Aeon Cell"
+		"Thermoelectric Cooler"
+		"Emergency Ramscoop"
+		"Research Laboratory"
+		"Salvage Scanner"
+		"Anvil-Class Engine"
+		"Crucible-Class Thruster"
+		"Forge-Class Thruster"
+		"Smelter-Class Thruster"
+		"Crucible-Class Steering"
+		"Forge-Class Steering"
+		"Smelter-Class Steering"
+
+mission "Remnant: Logistics Outfitter Delivery"
+	name "Outfitter Equipment Delivery"
+	description `Outfitter repair crews on <planet> need <cargo> parts delivered. Logistics adjustment is <payment>.`
+	cargo "remnant outfitter repair" 30 175
+	job
+	repeat
+	to offer
+			not "Remnant: Ssil Vida Active"
+			not "flagship planet: Ssil Vida"
+			has "Remnant: Logistics 1: done"
+			has "license: Remnant"
+			has "Remnant: Logistics Industrial Delivery: done" >= 9
+			random < 30
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A repair technician inspects the <cargo> parts and authorizes your payment of <payment>.`
+		payment 60000 20
+
+trade
+	commodity "remnant salvage"
+		"Builder salvage from the Graveyard"
+		"Ka'het Nullifier"
+		"Ka'het Ravager Beam"
+		"Ka'het Ravager Turret"
+		"Ka'het Annihilator"
+		"Ka'het Annihilator Turret"
+		"Annihilator Projectile"
+		"Ka'het EMP Deployer"
+		"Ka'het Emergency Deployer"
+		"Ka'het Support Cooling"
+		"Ka'het Primary Cooling"
+		"Ka'het Shield Restorer"
+		"Ka'het Grand Restorer"
+		"Ka'het MHD Generator"
+		"Ka'het Reserve Accumulator"
+		"Maeri Engine Nacelles"
+		"Ka'het Sustainer Nacelles"
+		"Telis Engine Nacelles"
+		"Vareti Engine Block"
+		"Ka'het Compact Engine"
+		"Candle class generator"
+		"Furnace class generator"
+		"Inferno class generator"
+		"Plasma Core"
+		"Double Plasma Core"
+		"Triple Plasma Core"
+		"small Systems Core"
+		"medium Systems Core"
+		"large Systems Core"
+		"Fuel Processor"
+		"Asteroid class thruster"
+		"Comet class thruster"
+		"Lunar class thruster"
+		"Planetary class steering"
+		"Asteroid class steering"
+		"Comet class steering"
+		"Lunar class steering"
+		"Planetary class steering"
```

Comment:
I'd _probably_ consider these all proper nouns and capitalize everything just to be safe. Did you have specific reasons for leaving some of these parts lowercase?

Reply:
I was trying to avoid the text being clumsy with the names of some of the Korath outfits in the outfits file:
outfit "Thruster (Asteroid Class)"
outfit "Thruster (Comet Class)"
outfit "Thruster (Lunar Class)"